code/modules/mob/mob.dm

```diff
@@ -371,8 +371,8 @@
 		var/deathtimeseconds = round((deathtime - deathtimeminutes * 600) / 10,1)
 		usr << "You have been dead for[pluralcheck] [deathtimeseconds] seconds."
 
-		if (deathtime < 18000)
-			usr << "You must wait 30 minutes to respawn!"
+		if (deathtime < 12000)

```

Comment:
Same as above, this is the least invasive edit I could possibly make, worst case Github reverts it right back during a merge.

Considering the state of Baycode, which we're currently tied to, Urist is currently working out the policy as to what to do, and one thing's for sure that after handling a good part of the code side of merging two Bay dev cycles at once I do *not* want to introduce another bulletpoint on the TODO list for the next one if possible.

I could macro it on top of this file, but it's literally only used here anyway.